src/rpcblockchain.cpp

```diff
@@ -384,6 +384,7 @@ UniValue getblock(const UniValue& params, bool fHelp)
             "  \"nonce\" : n,           (numeric) The nonce\n"
             "  \"bits\" : \"1d00ffff\", (string) The bits\n"
             "  \"difficulty\" : x.xxx,  (numeric) The difficulty\n"
+            "  \"chainwork\" : \"xxxx\",  (string) The estimated number of block header hashes checked from the genesis block to this block, in hexadecimal\n"
```

Comment:
What don't you understand about it specifically? Would:  

>(string) Expected number of hashes required to produce the chain up to this block (in hex)  

be better? As-is it's ambiguous to what "current chain" means.